StellarApp/Controllers/NewsViewController.swift

```diff
@@ -15,18 +16,27 @@ class NewsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.addSubview(newsView)
-        let firstColor = UIColor.init(red: 255/255, green: 0/255, blue: 204/255, alpha: 1)
-        let secondColor = UIColor.init(red: 51/255, green: 51/255, blue: 153/255, alpha: 1)
+        newsView.newsCollectionView.delegate = self
+        newsView.newsCollectionView.dataSource = self
+        getArticles(keyword: "astronomy")
+    }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return UIInterfaceOrientationMask.portrait
+    }
+    
+    override func viewDidLayoutSubviews() {
+        setGradient()
+    }
+    func setGradient() {
```

Comment:
can be made `private`

Reply:
Is this gradient needed. looks like it's just gray

Reply:
yes the gradient is needed because once I take the gradient out, we have a black background which gives it an awkward look